### ISSUE TITLE
Fire Extinguishers should have an extremely mild cooling effect

### ISSUE THREAD

**Issue by celestialrage** [studio-minus/ppg-bugs-and-requests#686]
**Feature description**
Make the fire extinguisher apply a mild cooling effect.
**Feature purpose**
When extinguishing humans / wood items, they often re-ignite as they're still so hot that they can burst back into flames. I propose that the fire extinguisher should not only remove fire, but apply a subtle cooling effect so that things don't reignite after being extinguished.

**Additional context**
N/A

**Comment by UniDuki** [comment]
I disagree, I think the fire extinguisher is already too op, one short little spray and all fires in front of it are put out. It already cools things that are on fire, making it cool object when not on fire just seems weird to me.

**Comment by celestialrage** [comment]
> I disagree, I think the fire extinguisher is already too op, one short little spray and all fires in front of it are put out. It already cools things that are on fire, making it cool object when not on fire just seems weird to me.


I think the argument of "OP" is a moot point when its a sandbox creative game with no goals or resource gathering. extinguishers IRL are usually CO2 which, cools stuff down. When you spray something IRL with an extinguisher, the flames go out immediately and it doesnt re-ignite. This is a small QOL change that I feel is more accurate to IRL.

**Comment by StupidRepo** [comment]
Alternatively, I could try to make a mod for this.
